pkg/conn/conn.go

```diff
@@ -32,69 +23,29 @@ import (
 	"google.golang.org/grpc/keepalive"
 
 	"github.com/pingcap/br/pkg/glue"
+	"github.com/pingcap/br/pkg/pdutil"
 	"github.com/pingcap/br/pkg/utils"
 )
 
 const (
-	dialTimeout          = 30 * time.Second
-	clusterVersionPrefix = "pd/api/v1/config/cluster-version"
-	regionCountPrefix    = "pd/api/v1/stats/region"
-	schdulerPrefix       = "pd/api/v1/schedulers"
-	maxMsgSize           = int(128 * utils.MB) // pd.ScanRegion may return a large response
-	scheduleConfigPrefix = "pd/api/v1/config/schedule"
+	dialTimeout = 30 * time.Second
 
 	resetRetryTimes = 3
 )
 
 // Mgr manages connections to a TiDB cluster.
 type Mgr struct {
-	pdClient pd.Client
-	pdHTTP   struct {
-		addrs []string
-		cli   *http.Client
-	}
-	tlsConf  *tls.Config
-	dom      *domain.Domain
-	storage  tikv.Storage
-	grpcClis struct {
+	PdController *pdutil.PdController
```

Comment:
could we make it an anonymous field?

```suggestion
	*pdutil.PdController
```